src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -162,7 +162,7 @@ String computeCrc32c(String input) {
         return Long.toHexString(crc.getValue());
     }
 
-    private static final int SLEEP_MILLISECONDS = 1000;
+    private static final int SLEEP_BASE_MILLISECONDS = 1000;
```

Comment:
Given the size of the value it would make more sense to use seconds here instead of milliseconds.

Reply:
I figure that milliseconds is what's traditionally used so I stuck with it but yeah, maybe I'll go back to turn to seconds it this helps things

Reply:
That's what's great about `TimeUnit`. We never have to mentally convert to seconds again.